ui/core.js

```diff
@@ -126,6 +128,67 @@ $.fn.extend( {
 				$( this ).removeAttr( "id" );
 			}
 		} );
+	},
+
+	form: function() {
```

Comment:
Is there a reason to not just do 
```js
return typeof this[ 0 ].form === "string" ? this.closest( "form" ) : $( this[ 0 ].form );
```

It avoids the small function call over head removes a line and is still under 100 chars?